embed-code-go/embedding/parsing/code_sample_line.go

```diff
@@ -36,14 +36,15 @@ type CodeSampleLine struct{}
 //
 // context — a context of the parsing process.
 func (c CodeSampleLine) Recognize(context ParsingContext) bool {
-	return !context.ReachedEOF() && context.codeFenceStarted
+	return !context.ReachedEOF() && context.CodeFenceStarted
 }
 
 // Moves to the next line.
 //
 // context — a context of the parsing process.
 //
 // config — a configuration of the embedding.
-func (c CodeSampleLine) Accept(context *ParsingContext, config configuration.Configuration) {
+func (c CodeSampleLine) Accept(context *ParsingContext, config configuration.Configuration) error {
```

Comment:
Please review all methods which signature has changed, and update their descriptions accordingly.